target/linux/ath79/dts/qca9558_ubnt_powerbeam-5ac-500.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "qca955x_ubnt_xc.dtsi"
+
+/ {
+	compatible = "ubnt,powerbeam-5ac-500", "ubnt,xc", "qca,qca9558";
+	model = "Ubiquiti PowerBeam 5AC 500";
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
```

Comment:
Maybe move that to DTSI, or do you have a reason for putting it here?

Reply:
Moving it to DTSI.